src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/CalledLegacySegmentCollection.java

```diff
@@ -0,0 +1,107 @@
+    /**
+     * Sample, Chromosome, Start, End, Num_Probes, Call, Segment_Mean
+     */
+    enum CalledLegacySegmentTableColumn {
+        SAMPLE("Sample"),
+        CHROMOSOME("Chromosome"),
+        START("Start"),
+        END("End"),
+        NUM_PROBES("Num_Probes"),
+        CALL("Call"),
+        SEGMENT_MEAN("Segment_Mean");
+
+        private final String columnName;
+
+        CalledLegacySegmentTableColumn(final String columnName) {
+            this.columnName = columnName;
+        }
+
+        static final TableColumnCollection COLUMNS = new TableColumnCollection(
+                Stream.of(values()).map(c -> c.columnName).toArray());
+    }
+
+    private static final Function<DataLine, CalledLegacySegment> LEGACY_SEGMENT_DATA_LINE_TO_RECORD_FUNCTION = dataLine -> {
+        final String sampleName = dataLine.get(CalledLegacySegmentTableColumn.SAMPLE.columnName);
+        final String contig = dataLine.get(CalledLegacySegmentTableColumn.CHROMOSOME.columnName);
+        final int start = dataLine.getInt(CalledLegacySegmentTableColumn.START.columnName);
+        final int end = dataLine.getInt(CalledLegacySegmentTableColumn.END.columnName);
+        final int numProbes = dataLine.getInt(CalledLegacySegmentTableColumn.NUM_PROBES.columnName);
+        final double segmentMean = dataLine.getDouble(CalledLegacySegmentTableColumn.SEGMENT_MEAN.columnName);
+        final String callOutputString = dataLine.get(CalledCopyRatioSegmentCollection.CalledCopyRatioSegmentTableColumn.CALL);
+        final CalledCopyRatioSegment.Call call = Arrays.stream(CalledCopyRatioSegment.Call.values())
+                .filter(c -> c.getOutputString().equals(callOutputString)).findFirst().orElse(null);;
+        final SimpleInterval interval = new SimpleInterval(contig, start, end);
+        return new CalledLegacySegment(sampleName, interval, numProbes, segmentMean, call);
+    };
+
+    private static final BiConsumer<CalledLegacySegment, DataLine> LEGACY_SEGMENT_RECORD_AND_DATA_LINE_BI_CONSUMER = (calledLegacySegment, dataLine) ->
+            dataLine.append(calledLegacySegment.getSampleName())
+                    .append(calledLegacySegment.getContig())
+                    .append(calledLegacySegment.getStart())
+                    .append(calledLegacySegment.getEnd())
+                    .append(calledLegacySegment.getNumProbes())
+                    .append(calledLegacySegment.getCall().getOutputString())
+                    .append(formatDouble(calledLegacySegment.getSegmentMean()));
+
+    public CalledLegacySegmentCollection(final SampleLocatableMetadata metadata,
+                                         final List<CalledLegacySegment> LegacySegments) {
+        super(metadata, LegacySegments, CalledLegacySegmentTableColumn.COLUMNS, LEGACY_SEGMENT_DATA_LINE_TO_RECORD_FUNCTION, LEGACY_SEGMENT_RECORD_AND_DATA_LINE_BI_CONSUMER);
+    }
+
+    public CalledLegacySegmentCollection(final CalledCopyRatioSegmentCollection collection) {
```

Comment:
Let's just do this construction/conversion with private static methods (or even just do it inline) in the `CallCopyRatioSegments` tool class.  No reason to couple these two collection classes, even if minimally, and the presence of a `convert` method in this class is idiosyncratic---one might wonder why there isn't a corresponding `convert` method in `CalledSegmentCollection` or corresponding methods in other classes for other possible conversions.

If you choose to keep these as methods over in the tool class, then the parameter names `segments` and `segment` would be more conventional than `collection` and `seg`.

Reply:
Done